bindings/capacitorjs/IotaWalletMobile.podspec

```
@@ -0,0 +1,25 @@
+
+  Pod::Spec.new do |s|
+    s.name = 'IotaWalletMobile'
+    s.version = '0.0.1'
+    s.summary = 'Capacitor plugin binding to the wallet library'
+    s.license = { :type => 'Apache-2.0' }
+    s.homepage = 'https://github.com/iotaledger/wallet.rs/bindings/capacitorjs/#readme'
```

We moved some things around, should be 
```suggestion
    s.homepage = 'https://github.com/iotaledger/wallet.rs/wallet/bindings/capacitorjs/#readme'
``` 
after rebase

---

This was already fixed in the original PR